features/e2e_tests/e2e_card_payments_tests.feature

```diff
@@ -26,3 +26,11 @@ Feature: E2E Card Payments
       | card                    |
       | VISA_STEP_UP_CARD       |
       | MASTERCARD_STEP_UP_CARD |
+
+  @e2e_config_bypass_mastercard
+  Scenario: Unsuccessful payment with bypassCard using Mastercard - invalid expiration date
+    When User fills payment form with defined card MASTERCARD_INVALID_EXP_DATE_CARD
+    And User clicks Pay button
+    Then User will see payment status information: "Invalid field"
+    And User will see that notification frame has "red" color
+    And User will see that "EXPIRATION_DATE" field is highlighted
```

Comment:
You can also add this step:    
 And User will see "Invalid field" message under field: "EXPIRATION_DATE"

Reply:
done